### DISCUSSION TITLE
Would you like to hide the footer and the header of Insomnia?

### DISCUSSION THREAD

**Discussion by subnetmarco** [Kong/insomnia#7468] [Polls]
Recently we have introduced the ability to hide the organizations tab, as you can see in the video:

📺 https://www.loom.com/share/275e44ba6d354a9a920130f4db6ccade

Would you like us to extend this capability to hide more things?

**Comment by aaronhmiller** [comment]
The option to reduce visual noise always a welcome choice!

**Comment by CurryYangxx** [comment]
https://github.com/user-attachments/assets/0a1016f3-dca7-4003-b2b8-56936b13a699

In V11, we add a new toggle button to hide the app header.
